browser/js/catalog/product.html

```diff
@@ -0,0 +1,44 @@
+<div class = "container-fluid">
```

Comment:
I like the spacing of the code here. 

Let's use this as a template for how to space out code so it's easily readable.